packages/components/app/styles/components/tabs.scss

```diff
@@ -77,19 +47,23 @@ $indicator-height: 3px;
       color: var(--token-color-foreground-action);
     }
   }
-}
 
-.hds-tabs__tab-link {
-  @include hds-focus-ring-with-pseudo-element($top: 4px, $right: 6px, $bottom: 4px, $left: 6px);
-  display: block;
-  padding: 8px 12px;
-  color: var(--token-color-foreground-primary);
-  white-space: nowrap;
-  text-decoration: none;
-  border-radius: var(--token-form-control-border-radius);
+  .hds-tabs__tab-link {
+    @include hds-focus-ring-with-pseudo-element($top: 4px, $right: 6px, $bottom: 4px, $left: 6px);
+    display: flex;
+    align-items: center;
+    height: 36px;
+    padding: 8px 12px;
```

Comment:
in _theory_, since you're setting a fixed vertical height and we don't allow the text to wrap on multiple lines, there's no need for vertical padding; that said, it's not a problem per se (but generally I prefer to omit it, in case in the future I need to change the `height`, I also have to remember to check the `padding`)

Reply:
Alight. I left it since it didn't make a difference but I can change it.